src/pages/library/DataBrowser.js

```diff
@@ -221,11 +225,30 @@ const Browser = () => {
   const [selectedData, setSelectedData] = useState([])
   const [requestDatasetAccessList, setRequestDatasetAccessList] = useState()
   const { dataCatalog, loading } = useDataCatalog()
+  const acknowledged = Utils.useStore(acknowledgmentStore)
 
   const toggleSelectedData = data => setSelectedData(_.xor([data]))
 
   return h(FooterWrapper, { alwaysShow: true }, [
     libraryTopMatter('browse & explore'),
+    !acknowledged && div({
```

Comment:
Check on the id within the store
```suggestion
    !acknowledged[id] && div({
```